sickbeard/helpers.py

```diff
@@ -962,7 +958,7 @@ def anon_url(*url):
     """
     Return a URL string consisting of the Anonymous redirect URL and an arbitrary number of values appended.
     """
-    return '' if None in url else '%s%s' % (sickbeard.ANON_REDIRECT, ''.join(str(s) for s in url))
+    return '' if None in url else '{0}{1}'.format(sickbeard.ANON_REDIRECT, ''.join(str(s) for s in url))
```

Comment:
Lines like this though you can't do the `!s` change to eliminate the `str` or it will fail since it has to be a string to `.join`